contrib/msvc/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 sridharb1
```

Comment:
```suggestion
Copyright (c) 2020 Sridhar Boovaraghavan
```

Reply:
I've modified this to be:  
```
Copyright (c) 2020 Sridhar Boovaraghavan  GitHub Handle: sridharb1
```
@sridharb1 Please defend your change, provide alternative or approve my version.